### DISCUSSION TITLE
Immich podman systemd deployment files

### DISCUSSION THREAD

**Discussion by jbtrystram** [immich-app/immich#7977] [General]
Hi ! 

I am a user of podman and I usually set up services files to host my containers so they can be managed as systemd services.

I created a set of units files to deploy immich within a pod, so I thought some people may make use of it. 
https://github.com/jbtrystram/immich-podman-systemd

Right now it is a bit clunky as podman 5 is not out yet and some feature will make it neater.

Would you be interested in merging that upstream ? If so let me know and I'll open a PR when it's more stable (it's still a bit fragile)

**Comment by bo0tzz** [comment]
Related: #1912

There are many different deployment methods around, I think at the moment we're not interested in upstreaming (and supporting) any more. Perhaps in the future when Immich is more stable we might reconsider.
